src/locales/en/constructor.json

```diff
@@ -0,0 +1,19 @@
+{
+  "titles": {
+    "model": "Model",
+    "basis": "Basis",
+    "pattern": "Pattern",
+    "bottom": "Bottom",
+    "size": "Size",
+    "defaultPrice": "Default item price:",
+    "totalPrice": "Total item price:",
+    "endPrice": "End price:",
+    "defaultPriceValue": "50 ",
```

Comment:
Cant find usage of "defaultPriceValue"

Reply:
in that case, I'll remove this file and spread translations into "common.json" and "buttons.json" with removing redundant translatons